src/main/java/at/hannibal2/skyhanni/features/gui/customscoreboard/ScoreboardEvents.kt

```diff
@@ -397,143 +365,117 @@ private fun getActiveEventLine(): List<String> {
     // but from other locations like the scoreboard
     val blockedEvents = listOf("Spooky Festival", "5th SkyBlock Anniversary")
     if (blockedEvents.contains(currentActiveEvent.removeColor())) return emptyList()
-
-    val currentActiveEventTime = TabListData.getTabList().firstOrNull { SbPattern.eventTimeEndsPattern.matches(it) }
-        ?.let {
-            SbPattern.eventTimeEndsPattern.matchMatcher(it) {
-                group("time")
-            }
-        }
+    val currentActiveEventTime = SbPattern.eventTimeEndsPattern.firstMatcher(TabListData.getTabList()) {
+        group("time")
+    } ?: "§cUnknown"
 
     return listOf(currentActiveEvent, " Ends in: §e$currentActiveEventTime")
 }
 
-private fun getActiveEventShowWhen(): Boolean =
-    getTablistEvent() != null && TabListData.getTabList().any { SbPattern.eventTimeEndsPattern.matches(it) }
+private fun getActiveEventShowWhen(): Boolean = true
 
 private fun getSoonEventLine(): List<String> {
     val soonActiveEvent = getTablistEvent() ?: return emptyList()
-    val soonActiveEventTime = TabListData.getTabList().firstOrNull { SbPattern.eventTimeEndsPattern.matches(it) }
-        ?.let {
-            SbPattern.eventTimeStartsPattern.matchMatcher(it) {
-                group("time")
-            }
-        }
+    val soonActiveEventTime = SbPattern.eventTimeEndsPattern.firstMatcher(TabListData.getTabList()) {
```

Comment:
```suggestion
    val soonActiveEventTime = SbPattern.eventTimeStartsPattern.firstMatcher(TabListData.getTabList()) {
```
(Mistake present in the original code, but should be fixed.)